prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -199,28 +199,84 @@ groups:
 
-  - alert: HostMemoryUtilizationHigh
+  - alert: APodMemoryUtilizationHigh
     expr: vrops_hostsystem_memory_usage_percentage{vccluster=~".*management.*"} > 90
     for: 30m
     labels:
       severity: info
       tier: vmware
       service: compute
       context: "{{ $labels.hostsystem }} memory"
-      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
     annotations:
-      description: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
-      summary: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      description: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
+      summary: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
 
-  - alert: HostCPUUtilizationHigh
+  - alert: APodCPUUtilizationHigh
     expr: vrops_hostsystem_cpu_usage_average_percentage{vccluster=~".*management.*"} > 90
     for: 30m
     labels:
       severity: info
       tier: vmware
       service: compute
       context: "{{ $labels.hostsystem }} cpu"
-      meta: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      meta: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
     annotations:
-      description: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
-      summary: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      description: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
+      summary: "CPU utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
+
+  - alert: HostHighNumberOfPacketsDropped
+    expr: vrops_hostsystem_alert_info{alert_name="Host is experiencing high number of packets dropped"}
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
+      context: "{{ $labels.hostsystem }} packets dropped"
+      meta: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+      playbook: docs/devops/alert/vcenter
+      no_alert_on_absence: "true"
+    annotations:
+      description: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+      summary: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+
+  - alert: HostNicLinkFlappingAlert
+    expr: vrops_hostsystem_alert_info{alert_name=~"ESXi host has detected a link status.*"}
```

Comment:
Is the wildcard really needed? The other `alert_names` seem to match straight.

Reply:
Yes. This alert is s bit special. The alert name is: `alert_name="ESXi host has detected a link status "flapping" on a physical NIC"` 

Double quotes.

Reply:
Shall I do so: `alert_name="ESXi host has detected a link status /"flapping"/ on a physical NIC"` ?

Reply:
Yes, put in the whole expected name please.